.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- THIS INFORMATION IS MANDATORY - YOUR ISSUE WILL BE CLOSED IF IT IS MISSING. If you don't know your Rasa version, use `rasa --version`. 
+Please format any code or console output with three ticks ``` above and below.
+If you are asking a usage question (e.g. "How do I do xyz") please post your question on https://forum.rasa.com instead -->
+
+**Rasa version**:
+
+**Rasa SDK version**:
+
+**Python version**:
+
+**Operating system** (windows, osx, ...):
+
+**Issue**:
+
+
+
+**Error (including full traceback)**:
+```
+
+```
+
+**Command or request that led to error**:
+```
+
+```
+
+**Content of configuration file (config.yml)** (if relevant):
+```yml
+
+```
+
+**Content of domain file (domain.yml)** (if relevant):
+```yml
+
+```
```

Comment:
maybe we should also include "contents of actions.py" since its bound to actually be their problem like 80% of the time